Makefile

```diff
@@ -84,16 +122,23 @@ endif
 $(PREP_LINK_SCRIPT) : $(LINK_SCRIPT)
 	$(VERBOSE) $(CC) $(AFLAGS) -E -P -C -x c $< -o $@
 
-$(TARGET): $(OBJS) $(PREP_LINK_SCRIPT)
+$(TARGET): $(PFMLIB_ARCHIVE) $(OBJS) $(PREP_LINK_SCRIPT)
 	@echo "LD " $@
-	$(VERBOSE) $(LD) -T $(PREP_LINK_SCRIPT) -o $@ $^
+	$(VERBOSE) $(LD) -T $(PREP_LINK_SCRIPT) -o $@ --static $(OBJS) $(PFMLIB_LINKER_FLAGS)
```

Comment:
is the `--static` necessary? (here and below)

Reply:
hmm not really. I was trying out a bunch of things and this seems like leftover. I'll remove it.